pallets/automation-time/src/lib.rs

```diff
@@ -131,6 +150,8 @@ pub mod pallet {
 		PastTime,
 		/// The message cannot be empty.
 		EmptyMessage,
+		/// The provided_id cannot be empty
+		EmptyProvidedId,
```

Comment:
dupe provided id would just be a dupe task right?

Reply:
correct